server/src/service/daemon.rs

```diff
@@ -0,0 +1,191 @@
+//  org.freedesOnceCellktop.Secret.Service
+
```

Comment:
I meant renaming the folder `server`, not the module name btw

Reply:
Done :)